src/components/Editor/ItemEditor.vue

```diff
@@ -47,27 +47,34 @@
       ></icon-button>
 
       <!-- add item button -->
-      <icon-button
-        class="rounded-xl w-8 h-8 px-2 hidden bp-360:block"
-        :iconConfig="addItemIconConfig"
-        :buttonClass="addItemButtonClass"
-        @click="removeSelectedItemIndex"
-        v-tooltip.top="addItemButtonTooltip"
-        :disabled="isInteractionDisabled"
-        ariaLabel="add item"
-        data-test="addItem"
-      ></icon-button>
+      <!-- adding an enclosing span to show tooltip when disabled -->
+      <span v-tooltip="addItemButtonTooltip" tabindex="0">
+        <icon-button
+          class="rounded-xl w-8 h-8 px-2 hidden bp-360:block"
+          :iconConfig="addItemIconConfig"
+          :buttonClass="addItemButtonClass"
+          @click="removeSelectedItemIndex"
+          v-tooltip="addItemButtonTooltip"
```

Comment:
can remove this then? (do check for everywhere that you've added span if there is already a v-tooltip added and remove it)